esp/services/ws_access/ws_accessService.cpp

```diff
@@ -2162,6 +2162,572 @@ bool Cws_accessEx::onResourcePermissions(IEspContext &context, IEspResourcePermi
 
     return true;
 }
+    
+bool Cws_accessEx::onQueryViews(IEspContext &context, IEspQueryViewsRequest &req, IEspQueryViewsResponse &resp)
+{
+    try
```

Comment:
Does this and other new methods need to be wrapped in a version check?
        double version = context.getClientVersion();
        if (version >= 1.10)